packages/viz/src/components/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,432 @@
+          min: string | number;
+        }) => string | number);
+    /**
+     * Minimum value on the vertical axis.
+     * If no value is provided, the chart will automatically set a min value in order for all values to be equally distributed.
+     * Set this property to `min` to use the maximum data value.
+     */
+    minValue?:
+      | string
+      | number
+      | "min"
+      | ((obj: {
+          max: string | number;
+          min: string | number;
+        }) => string | number);
+  };
+  /** Columns to use to group the measures. */
+  series?: string | string[];
+  /** Tooltip options. */
+  tooltip?: {
+    /** Whether to show the tooltip or not. Defaults to `true`. */
+    show?: boolean;
+    /** Formatter for the value in the tooltip. */
+    valueFormatter?: (value?: string | number) => string;
+  };
+  /** Legend options. */
+  legend?: {
+    /**
+     * Whether to show the legend or not.
+     *
+     * If they are multiple series, the legend will appear by default. Otherwise, the legend will not be shown.
+     */
+    show?: boolean;
+  };
+  /** Formatter for the lines names used on the tooltips and legend. */
+  lineNameFormatter?: (value?: string) => string;
+  /** Strategy to use when there are empty cells. Defaults to `void`. */
+  emptyCellMode?: HvChartEmptyCellMode;
+  /** Whether the area under the lines should be filled. Defaults to `false`. */
+  area?: boolean;
+  /** Sets opacity of the filled area if `area` is true. Defaults to `0.5`. */
+  areaOpacity?: number;
+  /** Whether the chart is stacked. Defaults to `false`. */
+  stacked?: boolean;
+  /** Ranger slider options for the horizontal axis. */
+  horizontalRangeSlider?: {
+    /** Whether to show the ranger slider or not. Defaults to `false`. */
+    show?: boolean;
+  };
+}
+
+/**
+ * A line chart or line plot or line graph is a type of chart which displays information as a series of data points
+ * connected by straight line segments. It is a basic type of chart common in many fields.
+ */
+export const HvLineChart = ({
+  data,
+  series,
+  xAxis,
+  measures,
+  legend,
+  tooltip,
+  lineNameFormatter,
+  area = false,
+  stacked = false,
+  emptyCellMode = "void",
+  horizontalRangeSlider,
+  areaOpacity = 0.5,
+}: HvLineChartProps) => {
+  const { activeTheme, selectedMode, selectedTheme } = useTheme();
+  const { theme } = useVizTheme();
+
+  const chartRef = useRef<ReactECharts>(null);
+  const isMounted = useRef<boolean>(false);
+
+  const chartData = useMemo(() => {
+    let tableData: ColumnTable;
+    if (data instanceof internal.ColumnTable) {
+      tableData = data;
+    } else if (Array.isArray(data)) {
+      tableData = from(data);
+    } else {
+      tableData = table(data);
+    }
+
+    const measuresFields = Array.isArray(measures.fields)
+      ? measures.fields
+      : [measures.fields];
+
+    const agFunction = Array.isArray(measures.aggregation)
+      ? measures.aggregation
+      : [measures.aggregation];
```

Comment:
Sorry, my previous suggestion removed the ability to specify a single aggregation that applies to all measures.

```suggestion
      : measuresFields.map(() => measures.aggregation as HvChartAggregation);
```